### DISCUSSION TITLE
Cannot start from the basics

### DISCUSSION THREAD

**Discussion by msmith000** [mrlt8/docker-wyze-bridge#1328] [Q&A]
Hey there - 
 After configuring my API key and Key ID and attempting to sign in, I get the following (in wyzeLog.txt)

I am again returned to the login screen. I also put in a token from my Google Authenticator that I use for Wyze.
 
I need help in resolving this in order to get my Wyze cam V2 working... I don't want to have to crawl up on my garage to install a card for RSTP. 

Any assistance would be appreciated!
[wyzeLog.txt](https://github.com/user-attachments/files/16703242/wyzeLog.txt)

**Comment by mrlt8** [comment]
> [API] code='1000' msg='Invalid credentials, please check username, password, keyid or apikey' method=POST path=/api/user/login

Looks like there's an issue with your credentials or Developer API Key/ID. Can you double check your credentials or try generating a new key/id on the [wyze developer portal](https://developer-api-console.wyze.com/#/apikey/view)?

If you're using a `docker-compose.yml`, and your credentials have a `$` character, you will need to escape it with another `$` sign (e.g., `pa$$word` > `pa$$$$word`) .
